<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui</class>
 <widget class="QMainWindow" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>481</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Texture.Text Generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/ic_launcher.png</normaloff>:/icons/ic_launcher.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>25</width>
    <height>25</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="rdoBtn_Def_Settings">
           <property name="text">
            <string>Default Settings</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="rdoBtn_Adv_Settings">
           <property name="text">
            <string>Ad&amp;vanced Settings</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_Open">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Open</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_Select">
            <property name="text">
             <string>Select Image/Directory:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="rdoBtn_File">
            <property name="text">
             <string>Fi&amp;le</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QRadioButton" name="rdoBtn_Directory">
            <property name="text">
             <string>Directory</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <layout class="QHBoxLayout" name="horizontalLayout"/>
          </item>
          <item row="1" column="0" colspan="4">
           <widget class="QPlainTextEdit" name="plainTxt_directorypath">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>23</height>
             </size>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QPushButton" name="btn_File_Explorer">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_Settings">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>202</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>280</height>
          </size>
         </property>
         <property name="title">
          <string>Advanced Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_Tiling">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Tiling:</string>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_Alpha">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Alpha Hint:</string>
            </property>
            <property name="indent">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="4">
           <widget class="QComboBox" name="comboBox_Tiling">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Horizontal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Vertical</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Both</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_alpha">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Alpha</string>
            </property>
            <property name="indent">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2" colspan="4">
           <widget class="QComboBox" name="comboBox_Alpha">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Opaque</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Semi-Transparent</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Masked</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="checkBox_Alpha">
            <property name="text">
             <string>Enabled</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="6">
           <widget class="QGroupBox" name="gridGroupBox">
            <property name="title">
             <string>Normal Maps</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="1" column="4">
              <widget class="QCheckBox" name="checkBox_Flip_Green">
               <property name="text">
                <string>Normal Map Flip Green Channel</string>
               </property>
              </widget>
             </item>
             <item row="2" column="6">
              <widget class="QPushButton" name="btn_File_Explorer_Normal">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string>Open</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="6">
              <widget class="QPlainTextEdit" name="plainTxt_Normal">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>23</height>
                </size>
               </property>
               <property name="inputMethodHints">
                <set>Qt::ImhNone</set>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="4">
              <widget class="QCheckBox" name="checkBox_Normal_Enabled">
               <property name="text">
                <string>Enabled</string>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_Generate">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Generate!</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2" rowspan="3">
     <widget class="QGroupBox" name="groupBox_Status">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Status</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="1">
        <widget class="QTextEdit" name="console">
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1096</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionTerms_Agreements"/>
    <addaction name="actionAbout"/>
    <addaction name="actionChange_Log"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="actionTerms_Agreements">
   <property name="text">
    <string>&amp;Terms  Agreements</string>
   </property>
  </action>
  <action name="actionChange_Log">
   <property name="text">
    <string>Change Log</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
